package.json

```diff
@@ -60,6 +60,8 @@
     "nuxt": "^3.3.1",
     "nuxt-icon": "^0.3.3",
     "rehype-external-links": "^2.1.0",
+    "sitemap": "^7.1.1",
+    "ufo": "^1.1.2",
```

Comment:
ufo is already packed into nuxt, so it is available without additional installation.

```suggestion
```